FUZZING.md

```diff
@@ -80,3 +80,47 @@ If both sets of tests pass, refresh the fuzzer corpora with `refresh_ssl_corpora
 cd fuzz
 ./refresh_ssl_corpora.sh /path/to/fuzzer/mode/build /path/to/non/fuzzer/mode/build
 ```
+
+## Adding New Fuzz Test Targets
+When adding new functionality, adding new fuzz tests are important to provide additional testing and verification that we are correct.
+
+### Steps
+1. `NEW_FUNCTION='name_of_new_fuzzing_target'`
+2. `touch fuzz/${NEW_FUNCTION}.cc` \
+    Write fuzzing code for libfuzzer to parse in `fuzz/${NEW_FUNCTION}.cc`. 
+3. `mkdir fuzz/${NEW_FUNCTION}_corpus_raw` \
+   Import unit test files into the `fuzz/${NEW_FUNCTION}_corpus_raw` directory. These will be used to create our new fuzzing corpus.
```

Comment:
> Import unit test files...

Where are the unit test files originally from?

Reply:
I got the original unit tests from our current OCSP test files. I also converted the `ors` files to a hex format, so the `d2i_OCSP_Response` could correctly parse it. 
The current `fuzz/ocsp_corpus` was generated from a 30 minute fuzzing run on those unit tests.